packages/backend-for-frontend/src/model/errors.ts

```diff
@@ -166,6 +167,16 @@ export function tenantNotFound(tenantId: string): ApiError<ErrorCodes> {
   });
 }
 
+export function tenantBySelfcareIdNotFound(
+  selfcareId: string
+): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `Tenant with Selfcare ID ${selfcareId} not found`,
+    code: "tenantSelfcareNotFound",
```

Comment:
Align code and function name
```suggestion
    code: "tenantBySelfcareIdNotFound",
```